tests/test_build.py

```diff
@@ -507,6 +507,7 @@ def tests_patch_get_outputs():
 
 
 def tests_patch___new__():
+    from compilertools.build import get_build_compile_args
     """Test _patch___new__"""
```

Comment:
Please keep the doc-string line on top of the function.

Reply:
> Please keep the doc-string line on top of the function.

Thank you for your feedback. I have made the changes as requested.